### PR TITLE
SEO: Change titles, make product copy more user friendly, improve route names

### PR DESCRIPTION
**Pull request by pakaplace**
What I did
* Changed a fair bit of the product copy to be more user-friendly / SEO-advantageous
* Added descriptive titles (h1, h2) to each screen
* Changes routes to be more SEO-friendly (e.g. /covid-timeline-tracker) instead of /timeline)

Next steps:
* Reach out to fellow covid projects to get backlinks to our site. More Backlinks --> higher search ranking

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Navbar = ({ width }) => {
                 className={classes.space}
                 variant={width !== 'xs' ? 'h6' : 'body1'}
               >
-                COVID Trial Dash{' '}
+                COVID Trial Volunteer Dashboard{' '}
```
Comment (j10sanders) [inline src/components/Navbar/Navbar.jsx:55]: This is tough because it's not mobile friendly... Let's discuss

File: src/routes/index.js [hunk 2]
```diff
@@ -21,26 +21,28 @@ const Routes = () => {
...
       <Route
-        path={'/volunteer'}
+        path={'/volunteer-covid-trial'}
         render={() => (
           <Iframe url={howYouCanHelpUrl} title='How and Why to Volunteer' />
         )}
       />
       <Route
-        path={'/trialssummary'}
+        path={'/covid-trials-summary'}
```
Comment (j10sanders) [inline src/routes/index.js:45]: nice!

**Comment by j10sanders** [comment]
probably good to be rid of that package-lock.json lol

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (j10sanders) [reply src/components/Navbar/Navbar.jsx:55]: maybe we remove the title when it's mobile and just keep the logo (currently doing the opposite)

File: src/routes/index.js (hunk 2 above)
Comment (Nestyko) [reply src/routes/index.js:45]: I love this new routes names.

Could it all be nested inside `vaccines`? something like `/vaccines/coronavirus-timeline-tracker`

That to be able to match the new menu we are doing.

I just asking if it makes sense on SEO side, we can merge it like this, and then once the menu is done I can update the routes to work with that.

**Comment by Nestyko** [comment]
> probably good to be rid of that package-lock.json lol

yes please. let's stick to only using `yarn`

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Nestyko) [reply src/components/Navbar/Navbar.jsx:55]: we can have it like this, and on mobile only show either the logo or the text. We can discuss later, but i'm ok with this change for now.

**Comment by pakaplace** [comment]
package-lock was added to my branch then deleted

**Pull request merged**